recipes/playrho/all/patches/1.1.0-0001-fix-target-name.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/PlayRho/CMakeLists.txt b/PlayRho/CMakeLists.txt
+index 423a8ac..b0528ad 100644
+--- a/PlayRho/CMakeLists.txt
++++ b/PlayRho/CMakeLists.txt
+@@ -90,10 +90,9 @@ if(PLAYRHO_BUILD_SHARED)
+ 		${PLAYRHO_Rope_SRCS}
+ 		${PLAYRHO_Rope_HDRS}
+ 	)
+-	target_compile_features(PlayRho PUBLIC cxx_std_17)
++	target_compile_features(PlayRho_shared PUBLIC cxx_std_17)
+ 	set_target_properties(PlayRho_shared PROPERTIES
+ 		OUTPUT_NAME "PlayRho"
+-		CLEAN_DIRECT_OUTPUT 1
```

Comment:
From the [documentation](https://linux.die.net/man/1/cmakeprops) that I can find on `CLEAN_DIRECT_OUTPUT`, this appears like having it would help. OTOH, cmake.org itself doesn't seem to have [documentation](https://cmake.org/?s=CLEAN_DIRECT_OUTPUT) on this. So I'm hoping someone can shed more light into this change. Thanks!

Reply:
Reviewing the cmake source code revision history, I think I've seen the light now LOLs! Looks like this property was removed from cmake back in 2009 in favor of always doing its behavior. I've made a [stackoverflow](https://stackoverflow.com/questions/75087258/what-of-cmake-clean-direct-output-property) Q&A for this in case anyone else comes across this mystery as well.